wiki/versions/1.19/data/recipes/java/src_mojmaps/main/java/org/quiltmc/wiki/recipes/MyRecipe.java

```diff
@@ -0,0 +1,123 @@
+package org.quiltmc.wiki.recipes;
+
+import I;
```

Comment:
Not a valid import.

Reply:
Yeah, I know, nothing we can do. That's just loom (or maybe Mercury) messing up

Reply:
Shouldn't be fixed manually then? Otherwise, it wouldn't be compilable code.

Reply:
Yes, but it'll be messed up any time the translating task is re-ran. I'll try to make some system for marking files to not automatically translate